examples/convert_ieeg_to_bids.py

```diff
@@ -0,0 +1,170 @@
+"""
+===================================
+07. Convert iEEG data to BIDS format
+===================================
+
+In this example, we use MNE-BIDS to create a BIDS-compatible directory of EEG
+data. Specifically, we will follow these steps:
+
+1. Download some iEEG data from the MNE-Python API
+   `PhysioBank database <https://physionet.org/physiobank/database>`_.
+    `https://mne.tools/stable/auto_tutorials/misc/plot_ecog.html#sphx-glr-auto-tutorials-misc-plot-ecog-py`_.
+
+2. Load the data, extract information, and save in a new BIDS directory
+
+3. Check the result and compare it with the standard
+
+The iEEG data will be pretty similar to the EEG data with the addition of extra elements in
+the electrodes.tsv and coord_system.json files.
+"""
+
+# Authors: Adam Li <adam2392@gmail.com>
+# License: BSD (3-clause)
+
+import os
+import tempfile
+
+import mne
+import numpy as np
+from scipy.io import loadmat
+
+from mne_bids import write_raw_bids, make_bids_basename, read_raw_bids
+from mne_bids.utils import print_dir_tree
+
+# need to install visualization
+from mpl_toolkits import mplot3d
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Step 1: Download the data
+# -------------------------
+#
+# First, we need some data to work with. We will use the
+# data downloaded via MNE-Python's API.
+#
+# `https://mne.tools/stable/generated/mne.datasets.misc.data_path.html#mne.datasets.misc.data_path`_.
+#
+# Conveniently, there is already a data loading function available with
+# MNE-Python:
+
+# Make a directory to save the data to
+home = os.path.expanduser('~')
+mne_dir = os.path.join(home, 'mne_data')
+if not os.path.exists(mne_dir):
+    os.makedirs(mne_dir)
```

Comment:
use

`mne.get_config('MNE_DATA')`

Reply:
we have this in another example that @sappelhoff wrote. That should be fixed too then.

Reply:
indeed, it'd be great to fix it here as well: https://github.com/mne-tools/mne-bids/blob/5ff266d4ba42095055fca5cc1ca7244cba416e74/examples/convert_eeg_to_bids.py#L50-L54